eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/ClusteredVariantsRestControllerTest.java

```diff
@@ -561,10 +558,62 @@ public void testGetMergedAndDeprecatedVariants()
         ResponseEntity<List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>>> response =
                 testRestTemplate.exchange(getVariantUrl, HttpMethod.GET, null, new ClusteredVariantType());
 
+        // then
+        assertEquals(HttpStatus.GONE, response.getStatusCode());
+        assertEquals(1, response.getBody().size());
+        assertEquals(clusteredVariantEntity3.getModel(), response.getBody().get(0).getData());
+        assertClusteredVariantCreatedDateNotNull(response.getBody());
+    }
+
+    @Test
+    public void testGetDeprecatedDbsnpClusteredVariant()
+            throws AccessionCouldNotBeGeneratedException, AccessionMergedException, AccessionDoesNotExistException,
+                   AccessionDeprecatedException {
+        // given
+        clusteredService.deprecate(DBSNP_CLUSTERED_VARIANT_ACCESSION_1, "deprecated for testing");
+        String getVariantUrl = URL + DBSNP_CLUSTERED_VARIANT_ACCESSION_1;
+
+        // when
+        ResponseEntity<List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>>> response =
+                testRestTemplate.exchange(getVariantUrl, HttpMethod.GET, null, new ClusteredVariantType());
+
         // then
         assertEquals(HttpStatus.GONE, response.getStatusCode());
         assertEquals(1, response.getBody().size());
         assertEquals(clusteredVariantEntity1.getModel(), response.getBody().get(0).getData());
         assertClusteredVariantCreatedDateNotNull(response.getBody());
     }
+
+    @Test
+    public void testGetDeprecatedEvaClusteredVariant()
+            throws AccessionCouldNotBeGeneratedException, AccessionMergedException, AccessionDoesNotExistException,
+                   AccessionDeprecatedException {
+        // given
+        Long deprecatedAccession = 1L;
+        ClusteredVariant variant1 = new ClusteredVariant("ASMACC01", 2000, "CHROM1", 1234, VariantType.SNV, false,
+                                                         null);
+        DbsnpClusteredVariantEntity clusteredVariantEntity1 = new DbsnpClusteredVariantEntity(deprecatedAccession,
+                                                                                              "hash-100", variant1, 1);
+        ClusteredVariant variant2 = new ClusteredVariant("ASMACC02", 2000, "CHROM2", 1234, VariantType.SNV, false,
+                                                         null);
+        Long otherAccession = 1L;
```

Comment:
If this is going to be the same accession, I think it would be more clear to use the same variable. If it not supposed to be the same accession and the test still passes, then it needs some checking.

Reply:
the reason why it still passed is that the `clusteredService.deprecate(...` call in this test was adding a second inactiveObject in the operation document, which was ignored, as every operation document in eva-accession has only one element (either for EVA or dbSNP, submitted or clustered variants).